src/status_im/ui/screens/browser/views.cljs

```diff
@@ -24,8 +24,14 @@
    [status-im.utils.slurp :refer [slurp]]
    [status-im.utils.views :as views]))
 
+(def browser-config-edn
+  (slurp "./src/status_im/utils/browser_config.edn"))
+
 (def browser-config
-  (edn/read-string (slurp "./src/status_im/utils/browser_config.edn")))
+  (let [config (atom nil)]
```

Comment:
could elaborate on atom? what the purpose?

Reply:
caching of the result, probably simpler to wrap with `memoize`

Reply:
does it look better now?

Reply:
it's not about look. i just didn't notice that it's `def` not `defn`, but this looks better :)